core/tools/blueos_startup_update/blueos_startup_update.py

```diff
@@ -263,6 +282,55 @@ def update_i2c4_symlink() -> bool:
     return False  # This patch doesn't require restart to take effect
 
 
+def revert_update_dwc2() -> bool:
+    """
+    Removes dwc2 configuration from cmdline.txt
+    This was being wrongly applied on Pi3 due to a bad host_cpu check.
+    """
+
+    # Remove dwc2 module configuration from cmdline
+    unpatched_cmdline_content = load_file(cmdline_file).replace("\n", "").split(" ")
+    cmdline_content = []
+    for item in unpatched_cmdline_content:
+        if "dwc2" not in item and "g_ether" not in item:
+            cmdline_content.append(item)
+
+    # Save if needed, with backup
+    if unpatched_cmdline_content == cmdline_content:
+        return False
+    save_file(cmdline_file, " ".join(cmdline_content), "before_revert_update_dwc2")
+
+    # Patch applied and system needs to be restarted for it to take effect
+    return True
```

Comment:
I know I'm the one who wrote this, but for the record, this will drop all of `modules_load` if either `dwc2` or `g_ether` is in there.
I think it is not worth the extra effort/code, as if someone manually add/remove a module, we can assume the offending modules are not there.

Reply:
yeah, the most correct would be to implement boot_cmdline_remove_module